### PR TITLE
Multiple arguments `by` option

### PR DESCRIPTION
**Pull request by dmdashenkov**

### CODE REVIEW

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/enrichments/EnrichmentsFinder.java [hunk 1]
```diff
@@ -186,26 +199,43 @@ private void putEntry(Multimap<String, String> targetMap, DescriptorProto msg) {
...
-                    log().debug("Skipping a wildcard event");
-                    // Ignore the wildcard By options, as we don't know the target event type in this case.
-                    continue;
-                }
-                if (eventNameFromBy == null) {
-                    throw invalidByOptionValue(msgName);
-                }
-                final String enrichmentName = packagePrefix + msgName;
-                return new AbstractMap.SimpleEntry<>(enrichmentName, eventNameFromBy);
+            if (hasOptionEnrichBy(field)) { // TODO:19-01-17:dmytro.dashenkov: Verify behavior. By now we stop searching after having found one appropriate match, but no we check all the fields.
```
Comment (dmdashenkov) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/enrichments/EnrichmentsFinder.java]: As for me, this behavior seems to be more consistent, but still this should be checked.

**Comment by dmdashenkov** [comment]
@armiol PTAL

File: enrichment-lookup-sample/src/test/java/org/spine3/gradle/protobuf/lookup/enrichments/EnrichmentLookupPluginShould.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ * @author Dmytro Dashenkov
+ */
+public class EnrichmentLookupPluginShould {
+
+    private static final Properties prop = new Properties();
+
+    @BeforeClass
+    public static void setUp() {
+        try {
+            final File propFile = new File("generated/test/resources/enrichments.properties");
```
Comment (armiol) [inline enrichment-lookup-sample/src/test/java/org/spine3/gradle/protobuf/lookup/enrichments/EnrichmentLookupPluginShould.java]: Let's move it out of `generated` folder. It should belong to the `test` sources of the same module as the test itself.
Also, it's better to load via the classloader's `getResourceAsStream(...)`.

File: enrichment-lookup-sample/src/test/proto/spine/test/plugin/enrichment.proto [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+package spine.test.plugin;
+
+option (type_url_prefix) = "type.spine3.org";
+option java_package="org.spine3.test.plugin";
+option java_multiple_files = true;
+
+import "spine/annotations.proto";
+import "spine/enrichment_annotations.proto";
+
+// Messages in this file are test enrichments
```
Comment (armiol) [inline enrichment-lookup-sample/src/test/proto/spine/test/plugin/enrichment.proto]: Dot at the end.

**Comment by armiol** [comment]
@dmdashenkov please also check the code fits 120-symbols-per-row.

File: enrichment-lookup-sample/src/test/java/org/spine3/gradle/protobuf/lookup/enrichments/EnrichmentLookupPluginShould.java (hunk 2 above)
Comment (dmdashenkov) [inline enrichment-lookup-sample/src/test/java/org/spine3/gradle/protobuf/lookup/enrichments/EnrichmentLookupPluginShould.java]: - The `enrichments.properties` is generated by the plugin, so I suppose it should stay in the `generated` directory. At least, that's how the plugin behaves by default.
 - By the reason unknown, the classloader is not able to find the file. I suppose that it's connected to the fact they don't belong to one directory (`test/...` vs `generated/...`).

**Comment by dmdashenkov** [comment]
@armiol BTW, what version do we give to this patch?

**Comment by armiol** [comment]
@dmdashenkov please use the version, which the `core-java` has at the moment.

**Comment by dmdashenkov** [comment]
Version set to `0.7.24-SNAPSHOT`.

**Comment by dmdashenkov** [comment]
@armiol PTAL again

**Pull request merged**